app/views/govuk_component/docs/tasklist.yml

```diff
@@ -0,0 +1,342 @@
+name: Tasklist
+description: Numbered steps containing expanding/collapsing sections
+body: |
+  Tasklists are designed to show a sequence of steps towards a specific goal, such as 'learning to drive'. Each step of a tasklist can contain one or more sections. Each section can contain one or more links to pages. Each section is collapsed by default so that users are not overwhelmed with information.
```

Comment:
Should we specifically refer to the research about why task lists are collapsed?

Reply:
Will add.